src/services/extensions/exposedComponents.tsx

```diff
@@ -0,0 +1,27 @@
+import { LinkButton } from '@grafana/ui';
+import { OpenInExploreLogsButtonProps } from 'Components/OpenInExploreLogsButton/types';
+import React, { lazy, Suspense } from 'react';
+const OpenInExploreLogsButton = lazy(() => import('Components/OpenInExploreLogsButton/OpenInExploreLogsButton'));
+
+function SuspendedOpenInExploreLogsButton(props: OpenInExploreLogsButtonProps) {
+  return (
+    <Suspense
+      fallback={
+        <LinkButton variant="secondary" disabled>
+          Open in Explore logs
+        </LinkButton>
+      }
+    >
+      <OpenInExploreLogsButton {...props} />
+    </Suspense>
+  );
+}
+
+export const exposedComponents = [
```

Comment:
Have you considered creating a single component that allows linking to any app that register an extension point instead of having single button specific to single app? Something similar to https://github.com/grafana/explore-profiles/blob/62720ad1cbd1346c772a521c133988d043851f55/src/links.ts#L42